src/Miunie.WindowsApp/Miunie.WindowsApp.csproj

```diff
@@ -149,7 +149,6 @@
     <AppxManifest Include="Package.appxmanifest">
       <SubType>Designer</SubType>
     </AppxManifest>
-    <None Include="Miunie.WindowsApp_TemporaryKey.pfx" />
```

Comment:
Is there a reason why you removed the TempKey? 🤔

Reply:
Not really. Never intended to do so lol

Reply:
You think you could add this back?

Reply:
Will be back in seconds.